src/controller/users_controls.py

```diff
@@ -0,0 +1,19 @@
+from fastapi.encoders import jsonable_encoder
+from src.utils.functions import model
+from ..schema.users_model import user
+from ..utils.auth import hash
+    
+async def register(User):
+    try:
+        User=user()
+        mail= User.email
+        check= await model.findone(mail,"questions")
```

Comment:
Why are we looking for user/searching for user with a question?

Reply:
resolved